types/module/server/manager.go

```diff
@@ -199,6 +205,21 @@ func (mm *Manager) InitGenesis(ctx sdk.Context, genesisData map[string]json.RawM
 	return res
 }
 
+// RunMigrations performs state migrations for registered modules.
+func (mm *Manager) RunMigrations(ctx sdk.Context, cdc codec.Codec) error {
+	for _, h := range mm.migrationHandlers {
```

Comment:
This needs to be sorted to run deterministically... maybe at this point we should just use the SDK's `AppModule` to not risk these kinds of issues

Reply:
There's only one handler, which is for the ecocredit module. In that case, is this still an issue?

Reply:
Best to just do it now and not have to worry